src/screens/Profiles/components/AvatarHeader/component.js

```diff
@@ -25,11 +25,13 @@ AvatarHeader.propTypes = {
   title: PropTypes.string,
   subtitle: PropTypes.string,
   sideElement: PropTypes.element,
+  boxed: PropTypes.bool,
 };
 
 AvatarHeader.defaultProps = {
   avatar: null,
   title: '',
   subtitle: '',
   sideElement: null,
+  boxed: false,
```

Comment:
What about making `boxed: undefined` and `numberOfLines={boxed && 1}` or something similar.